src/components/ProductInfo.vue

```diff
@@ -0,0 +1,392 @@
+                <h4 class="panel-title product-accordion-title text-uppercase">
+                  <a id="pdp-delivery-returns-toggle"
+                     class="collapsed pdp-accord-toggle"
+                     data-toggle="collapse"
+                     data-parent="#accordion-product-info"
+                     href="#collapseDelivery"
+                     aria-expanded="false true"
+                     aria-controls="collapseDelivery"
+                     @click="openAccordion">
+                  {{ $t('delivery.title') }}
+                    <img class="accordion-plus"
+                         src="../assets/img/plus79.png"
+                         alt="accordion content" />
+                  </a>
+                </h4>
+              </div>
+              <div id="collapseDelivery"
+                   class="panel-collapse collapse"
+                   role="tabpanel"
+                   aria-labelledby="headingDelivery">
+                <div class="panel-body panel-body-pdp">
+                  <ul class="product-delivery-list">
+
+                    <!-- {{#each deliveryRates.list}}
+                    <li>
+                      <span>{{name}}:</span> {{price}}{{#if freeAbove}} (FREE for orders above {{freeAbove}}){{/if}}
+                      {{#if description}}. {{description}}.{{/if}}
+                    </li>
+                    {{/each}} -->
+
+                    <li>{{ $t('delivery.freeReturns') }}</li>
+                    <li>{{ $t('delivery.moreDeliveryInfo') }}</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- Social media icons-->
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="social-share">
+            <ul class="social-icons pdp-social-icons">
+              <a href="">
+                <li>
+                  <img class="social-icon"
+                       src="../assets/img/Facebook.png"
+                       alt="facebook">
+                </li>
+              </a>
+              <a href="">
+                <li>
+                  <img class="social-icon"
+                       src="../assets/img/Pinterest.png"
+                       alt="pinterest">
+                </li>
+              </a>
+              <a href="">
+                <li>
+                  <img class="social-icon"
+                       src="../assets/img/Google.png"
+                       alt="google plus">
+                </li>
+              </a>
+            </ul>
+            <ul class="social-icons">
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+import ProductGallery from '@/components/ProductGallery.vue';
+import priceMixin from '@/mixins/priceMixin';
+
+export default {
+  components: {
+    ProductGallery,
+  },
+
+  data: () => ({
+    product: {},
+  }),
+
+  props: {
```

Comment:
I would put props before data because it's more important (at least it's more relevant to people using this component), but I'm not sure how we did it in other components...